osx/post_installation.sh

```diff
@@ -177,4 +179,12 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+((STEP++))
+echo "$STEP/$STEPS. Creating a KA-Lite_Uninstall.tool..."
+cp -R "$PRE_INSTALL_SCRIPT" "$APPLICATION_PATH/KA-Lite_Uninstall.tool"
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Error creating a KA-Lite_Uninstall.tool."
+    exit 1
+fi
+
```

Comment:
@arceduardvincent Can you put "KA-Lite_Uninstall.tool" into a variable so it's easy to maintain?